docs/src/pages/components/data-grid/components/CustomPaginationGrid.js

```diff
@@ -1,23 +1,15 @@
 import * as React from 'react';
-import { makeStyles } from '@mui/styles';
+import Box from '@mui/material/Box';
 import { DataGrid, useGridApiContext, useGridState } from '@mui/x-data-grid';
 import { useDemoData } from '@mui/x-data-grid-generator';
 import Pagination from '@mui/material/Pagination';
 
-const useStyles = makeStyles({
-  root: {
-    display: 'flex',
```

Comment:
Was this not needed?

Reply:
It does nothing. In other demos we don't use `display: 'flex'`.